com.hazelcast.jet.elastic.AuthElasticSinksTest and com.hazelcast.jet.elastic.AuthElasticSourcesTest [HZ-1275]

_master_ (commit e289a50b974b34a1696c851cfce626f94dd26069)

Failed on ZingJDK8: 
- http://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-master-ZingJDK8/423/testReport/com.hazelcast.jet.elastic/AuthElasticSinksTest/
- http://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-master-ZingJDK8/423/testReport/com.hazelcast.jet.elastic/AuthElasticSourcesTest/

Following tests failed:
- `com.hazelcast.jet.elastic.AuthElasticSinksTest.given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException`
- `com.hazelcast.jet.elastic.AuthElasticSinksTest.given_clientWithoutAuthentication_whenWriteToElasticSink_thenFailWithAuthenticationException`
- `com.hazelcast.jet.elastic.AuthElasticSourcesTest.given_clientWithWrongPassword_whenReadFromElasticSource_thenFailWithAuthenticationException`
- `com.hazelcast.jet.elastic.AuthElasticSourcesTest.given_clientWithoutAuthentication_whenReadFromElasticSource_then_failWithAuthenticationException`

<details><summary>Stacktrace (for com.hazelcast.jet.elastic.AuthElasticSinksTest.given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException as an example):</summary>

```
org.testcontainers.containers.ContainerLaunchException: Container startup failed
	at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:345)
	at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:326)
	at com.hazelcast.jet.elastic.ElasticSupport.lambda$static$1(ElasticSupport.java:54)
	at com.hazelcast.jet.impl.util.MemoizingSupplier.get(MemoizingSupplier.java:39)
	at com.hazelcast.jet.elastic.AuthElasticSinksTest.given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException(AuthElasticSinksTest.java:79)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:115)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:107)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.rnorth.ducttape.RetryCountExceededException: Retry limit hit with exception
	at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:88)
	at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:338)
	... 16 more
Caused by: org.testcontainers.containers.ContainerLaunchException: Could not create/start container
	at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:537)
	at org.testcontainers.containers.GenericContainer.lambda$doStart$0(GenericContainer.java:340)
	at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:81)
	... 17 more
Caused by: org.testcontainers.containers.ContainerLaunchException: Timed out waiting for log output matching '.*("message":\s?"started".*|] started
$)'
	at org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy.waitUntilReady(LogMessageWaitStrategy.java:49)
	at org.testcontainers.containers.wait.strategy.AbstractWaitStrategy.waitUntilReady(AbstractWaitStrategy.java:51)
	at org.testcontainers.containers.GenericContainer.waitUntilContainerStarted(GenericContainer.java:926)
	at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:480)
	... 19 more
```

</details>

<details><summary>Standard output (for com.hazelcast.jet.elastic.AuthElasticSinksTest.given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException as an example):</summary>

```
11:07:02,079  INFO || - [ImageNameSubstitutor] main - Image name substitution will be performed by: DefaultImageNameSubstitutor (composite of 'ConfigurationFileImageNameSubstitutor' and 'PrefixingImageNameSubstitutor')
11:07:02,213  INFO || - [DockerClientProviderStrategy] main - Loaded org.testcontainers.dockerclient.UnixSocketClientProviderStrategy from ~/.testcontainers.properties, will try it first
11:07:04,519  INFO || - [DockerClientProviderStrategy] main - Found Docker environment with local Unix socket (unix:///var/run/docker.sock)
11:07:04,522  INFO || - [DockerClientFactory] main - Docker host IP address is localhost
11:07:04,678  INFO || - [DockerClientFactory] main - Connected to docker: 
  Server Version: 20.10.5
  API Version: 1.41
  Operating System: Red Hat Enterprise Linux Server 7.9 (Maipo)
  Total Memory: 30514 MB
11:07:05,046  INFO || - [RegistryAuthLocator] main - Failure when attempting to lookup auth config. Please ignore if you don't have images in an authenticated registry. Details: (dockerImageName: testcontainers/ryuk:0.3.3, configFile: /home/ec2-user/.docker/config.json. Falling back to docker-java default behaviour. Exception message: /home/ec2-user/.docker/config.json (No such file or directory)
11:07:07,249  INFO || - [RyukResourceReaper] main - Ryuk started - will monitor and terminate Testcontainers containers on JVM exit
11:07:07,249  INFO || - [DockerClientFactory] main - Checking the system...
11:07:07,251  INFO || - [DockerClientFactory] main - ✔︎ Docker server version should be at least 1.6.0
11:07:07,437  INFO || - [DockerClientFactory] main - ✔︎ Docker environment should have more than 2GB free disk space
11:07:07,502  INFO || - [1]] main - Pulling docker image: elasticsearch:7.10.1. Please be patient; this may take some time but only needs to be done once.
11:07:07,710  INFO || - [1]] docker-java-stream-1560591884 - Starting to pull image
11:07:07,765  INFO || - [1]] docker-java-stream-1560591884 - Pulling image layers:  0 pending,  0 downloaded,  0 extracted, (0 bytes/0 bytes)
11:07:07,946  INFO || - [1]] docker-java-stream-1560591884 - Pulling image layers:  8 pending,  1 downloaded,  0 extracted, (611 KB/? MB)
11:07:08,133  INFO || - [1]] docker-java-stream-1560591884 - Pulling image layers:  7 pending,  2 downloaded,  0 extracted, (9 MB/? MB)
11:07:08,220  INFO || - [1]] docker-java-stream-1560591884 - Pulling image layers:  6 pending,  3 downloaded,  0 extracted, (14 MB/? MB)
11:07:08,297  INFO || - [1]] docker-java-stream-1560591884 - Pulling image layers:  5 pending,  4 downloaded,  0 extracted, (16 MB/? MB)
11:07:08,361  INFO || - [1]] docker-java-stream-1560591884 - Pulling image layers:  4 pending,  5 downloaded,  0 extracted, (19 MB/? MB)
11:07:08,471  INFO || - [1]] docker-java-stream-1560591884 - Pulling image layers:  3 pending,  6 downloaded,  0 extracted, (25 MB/? MB)
11:07:08,545  INFO || - [1]] docker-java-stream-1560591884 - Pulling image layers:  2 pending,  7 downloaded,  0 extracted, (34 MB/? MB)
11:07:10,232  INFO || - [1]] docker-java-stream-1560591884 - Pulling image layers:  1 pending,  8 downloaded,  0 extracted, (142 MB/? MB)
11:07:19,820  INFO || - [1]] docker-java-stream-1560591884 - Pulling image layers:  0 pending,  9 downloaded,  0 extracted, (378 MB/384 MB)
11:07:25,537  INFO || - [1]] docker-java-stream-1560591884 - Pulling image layers:  0 pending,  9 downloaded,  1 extracted, (382 MB/384 MB)
11:07:27,440  INFO || - [1]] docker-java-stream-1560591884 - Pulling image layers:  0 pending,  9 downloaded,  2 extracted, (382 MB/384 MB)
11:07:27,753  INFO || - [1]] docker-java-stream-1560591884 - Pulling image layers:  0 pending,  9 downloaded,  3 extracted, (382 MB/384 MB)
11:07:52,341  INFO || - [1]] docker-java-stream-1560591884 - Pulling image layers:  0 pending,  9 downloaded,  4 extracted, (384 MB/384 MB)
11:07:52,626  INFO || - [1]] docker-java-stream-1560591884 - Pulling image layers:  0 pending,  9 downloaded,  5 extracted, (384 MB/384 MB)
11:07:52,877  INFO || - [1]] docker-java-stream-1560591884 - Pulling image layers:  0 pending,  9 downloaded,  6 extracted, (384 MB/384 MB)
11:07:53,851  INFO || - [1]] docker-java-stream-1560591884 - Pulling image layers:  0 pending,  9 downloaded,  7 extracted, (384 MB/384 MB)
11:07:54,172  INFO || - [1]] docker-java-stream-1560591884 - Pulling image layers:  0 pending,  9 downloaded,  8 extracted, (384 MB/384 MB)
11:07:54,531  INFO || - [1]] docker-java-stream-1560591884 - Pulling image layers:  0 pending,  9 downloaded,  9 extracted, (384 MB/384 MB)
11:07:54,608  INFO || - [1]] docker-java-stream-1560591884 - Pull complete. 9 layers, pulled in 46s (downloaded 384 MB at 8 MB/s)
11:07:54,617  INFO || - [1]] main - Starting an elasticsearch container using [elasticsearch:7.10.1]
11:07:54,625  INFO || - [1]] main - Creating container for image: elasticsearch:7.10.1
11:07:58,528  INFO || - [1]] main - Container elasticsearch:7.10.1 is starting: fe559d2adf0b9114c6fdf69043204fb5ca5cde072f219945718d62d054019809
11:08:48,151  INFO || - [1]] main - Container elasticsearch:7.10.1 started in PT53.526900636S
11:08:51,899  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [MetricsConfigHelper] main - [LOCAL] [dev] [5.2-SNAPSHOT] Overridden metrics configuration with system property 'hazelcast.metrics.collection.frequency'='1' -> 'MetricsConfig.collectionFrequencySeconds'='1'
11:08:52,076  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [logo] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] 
	+       +  o    o     o     o---o o----o o      o---o     o     o----o o--o--o
	+ +   + +  |    |    / \       /  |      |     /         / \    |         |   
	+ + + + +  o----o   o   o     o   o----o |    o         o   o   o----o    |   
	+ +   + +  |    |  /     \   /    |      |     \       /     \       |    |   
	+       +  o    o o       o o---o o----o o----o o---o o       o o----o    o   
11:08:52,093  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [system] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
11:08:52,093  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [system] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Hazelcast Platform 5.2-SNAPSHOT (20220513 - e289a50) starting at [127.0.0.1]:5701
11:08:52,093  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [system] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Cluster name: dev
11:08:52,093  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [system] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Integrity Checker is disabled. Fail-fast on corrupted executables will not be performed. For more information, see the documentation for Integrity Checker.
11:08:52,094  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [system] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Jet is enabled
11:08:52,266  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [MetricsConfigHelper] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Collecting debug metrics and sending to diagnostics is enabled
11:08:53,991  WARN |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [CPSubsystem] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] CP Subsystem is not enabled. CP data structures will operate in UNSAFE mode! Please note that UNSAFE mode will not provide strong consistency guarantees.
11:08:55,598  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [JetServiceBackend] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Setting number of cooperative threads and default parallelism to 2
11:08:55,665  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [Diagnostics] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Diagnostics disabled. To enable add -Dhazelcast.diagnostics.enabled=true to the JVM arguments.
11:08:55,714  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [LifecycleService] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] [127.0.0.1]:5701 is STARTING
11:08:55,795  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [ClusterService] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] 

Members {size:1, ver:1} [
	Member [127.0.0.1]:5701 - 3612a7c2-a32a-4867-b5db-523200c1feca this
]

11:08:55,869  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [JobCoordinationService] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Jet started scanning for jobs
11:08:55,872 DEBUG |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [JobCoordinationService] hz.boring_johnson.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Not starting jobs because partitions are not yet initialized.
11:08:55,888  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [LifecycleService] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] [127.0.0.1]:5701 is STARTED
11:08:55,974 DEBUG |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [JobCoordinationService] hz.boring_johnson.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Not starting jobs because partitions are not yet initialized.
11:08:56,081  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [PartitionStateManager] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Initializing cluster partition table arrangement...
11:08:56,086 DEBUG |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [JobCoordinationService] hz.boring_johnson.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Not starting jobs because partitions are not yet initialized.
11:08:56,156  INFO || - [1]] given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException - Starting an elasticsearch container using [elasticsearch:7.10.1]
11:08:56,157  INFO || - [1]] given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException - Creating container for image: elasticsearch:7.10.1
11:08:56,202 DEBUG |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [JobRepository] hz.boring_johnson.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Job cleanup took 0ms
11:08:56,411  INFO || - [1]] given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException - Container elasticsearch:7.10.1 is starting: 68a4a5ec8af77a3c54dcbbe22c7546a5ee1d25257f1e87c484e0232c3ed3ad06
11:09:01,208 DEBUG |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [JobRepository] hz.boring_johnson.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Job cleanup took 0ms
11:09:06,210 DEBUG |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [JobRepository] hz.boring_johnson.cached.thread-4 - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Job cleanup took 0ms
11:09:11,212 DEBUG |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [JobRepository] hz.boring_johnson.cached.thread-1 - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Job cleanup took 0ms
11:09:15,945  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [HealthMonitor] hz.boring_johnson.HealthMonitor - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] processors=4, physical.memory.total=29.8G, physical.memory.free=8.0G, swap.space.total=0, swap.space.free=0, heap.memory.used=614.0M, heap.memory.free=2.7G, heap.memory.total=3.3G, heap.memory.max=3.3G, heap.memory.used/total=18.44%, heap.memory.used/max=18.44%, minor.gc.count=0, minor.gc.time=0ms, major.gc.count=0, major.gc.time=0ms, unknown.gc.count=4, unknown.gc.time=1859ms, load.process=0.00%, load.system=100.00%, load.systemAverage=12.25, thread.count=40, thread.peakCount=40, cluster.timeDiff=0, event.q.size=0, executor.q.async.size=0, executor.q.client.size=0, executor.q.client.query.size=0, executor.q.client.blocking.size=0, executor.q.query.size=0, executor.q.scheduled.size=0, executor.q.io.size=0, executor.q.system.size=0, executor.q.operations.size=0, executor.q.priorityOperation.size=0, operations.completed.count=1, executor.q.mapLoad.size=0, executor.q.mapLoadAllKeys.size=0, executor.q.cluster.size=0, executor.q.response.size=0, operations.running.count=0, operations.pending.invocations.percentage=0.00%, operations.pending.invocations.count=0, proxy.count=1, clientEndpoint.count=0, connection.active.count=0, client.connection.count=0, connection.count=0
11:09:16,216 DEBUG |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [JobRepository] hz.boring_johnson.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Job cleanup took 0ms
11:09:21,220 DEBUG |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [JobRepository] hz.boring_johnson.cached.thread-1 - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Job cleanup took 0ms
11:09:26,226 DEBUG |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [JobRepository] hz.boring_johnson.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Job cleanup took 0ms
11:09:31,231 DEBUG |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [JobRepository] hz.boring_johnson.cached.thread-3 - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Job cleanup took 0ms
11:09:35,951  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [HealthMonitor] hz.boring_johnson.HealthMonitor - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] processors=4, physical.memory.total=29.8G, physical.memory.free=6.6G, swap.space.total=0, swap.space.free=0, heap.memory.used=618.0M, heap.memory.free=2.6G, heap.memory.total=3.3G, heap.memory.max=3.3G, heap.memory.used/total=18.56%, heap.memory.used/max=18.56%, minor.gc.count=0, minor.gc.time=0ms, major.gc.count=0, major.gc.time=0ms, unknown.gc.count=4, unknown.gc.time=1859ms, load.process=0.00%, load.system=100.00%, load.systemAverage=12.88, thread.count=41, thread.peakCount=41, cluster.timeDiff=0, event.q.size=0, executor.q.async.size=0, executor.q.client.size=0, executor.q.client.query.size=0, executor.q.client.blocking.size=0, executor.q.query.size=0, executor.q.scheduled.size=0, executor.q.io.size=0, executor.q.system.size=0, executor.q.operations.size=0, executor.q.priorityOperation.size=0, operations.completed.count=1, executor.q.mapLoad.size=0, executor.q.mapLoadAllKeys.size=0, executor.q.cluster.size=0, executor.q.response.size=0, operations.running.count=0, operations.pending.invocations.percentage=0.00%, operations.pending.invocations.count=0, proxy.count=1, clientEndpoint.count=0, connection.active.count=0, client.connection.count=0, connection.count=0
11:09:36,243 DEBUG |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [JobRepository] hz.boring_johnson.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Job cleanup took 0ms
11:09:41,249 DEBUG |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [JobRepository] hz.boring_johnson.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Job cleanup took 0ms
11:09:46,252 DEBUG |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [JobRepository] hz.boring_johnson.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Job cleanup took 0ms
11:09:51,254 DEBUG |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [JobRepository] hz.boring_johnson.cached.thread-6 - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Job cleanup took 0ms
11:09:55,961  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [HealthMonitor] hz.boring_johnson.HealthMonitor - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] processors=4, physical.memory.total=29.8G, physical.memory.free=6.5G, swap.space.total=0, swap.space.free=0, heap.memory.used=622.0M, heap.memory.free=2.6G, heap.memory.total=3.3G, heap.memory.max=3.3G, heap.memory.used/total=18.68%, heap.memory.used/max=18.68%, minor.gc.count=0, minor.gc.time=0ms, major.gc.count=0, major.gc.time=0ms, unknown.gc.count=4, unknown.gc.time=1859ms, load.process=0.00%, load.system=0.00%, load.systemAverage=14.15, thread.count=41, thread.peakCount=41, cluster.timeDiff=0, event.q.size=0, executor.q.async.size=0, executor.q.client.size=0, executor.q.client.query.size=0, executor.q.client.blocking.size=0, executor.q.query.size=0, executor.q.scheduled.size=0, executor.q.io.size=0, executor.q.system.size=0, executor.q.operations.size=0, executor.q.priorityOperation.size=0, operations.completed.count=1, executor.q.mapLoad.size=0, executor.q.mapLoadAllKeys.size=0, executor.q.cluster.size=0, executor.q.response.size=0, operations.running.count=0, operations.pending.invocations.percentage=0.00%, operations.pending.invocations.count=0, proxy.count=1, clientEndpoint.count=0, connection.active.count=0, client.connection.count=0, connection.count=0
11:09:56,255 DEBUG |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [JobRepository] hz.boring_johnson.cached.thread-2 - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Job cleanup took 0ms
11:09:58,150 ERROR || - [1]] given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException - Could not start container
org.testcontainers.containers.ContainerLaunchException: Timed out waiting for log output matching '.*("message":\s?"started".*|] started
$)'
	at org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy.waitUntilReady(LogMessageWaitStrategy.java:49)
	at org.testcontainers.containers.wait.strategy.AbstractWaitStrategy.waitUntilReady(AbstractWaitStrategy.java:51)
	at org.testcontainers.containers.GenericContainer.waitUntilContainerStarted(GenericContainer.java:926)
	at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:480)
	at org.testcontainers.containers.GenericContainer.lambda$doStart$0(GenericContainer.java:340)
	at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:81)
	at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:338)
	at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:326)
	at com.hazelcast.jet.elastic.ElasticSupport.lambda$static$1(ElasticSupport.java:54)
	at com.hazelcast.jet.impl.util.MemoizingSupplier.get(MemoizingSupplier.java:39)
	at com.hazelcast.jet.elastic.AuthElasticSinksTest.given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException(AuthElasticSinksTest.java:79)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:115)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:107)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
11:09:58,249 ERROR || - [1]] given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException - Log output from the failed container:
Created elasticsearch keystore in /usr/share/elasticsearch/config/elasticsearch.keystore
{"type": "server", "timestamp": "2022-05-13T11:09:34,296Z", "level": "INFO", "component": "o.e.n.Node", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "version[7.10.1], pid[7], build[default/docker/1c34507e66d7db1211f66f3513706fdf548736aa/2020-12-05T01:00:33.671820Z], OS[Linux/3.10.0-1160.24.1.el7.x86_64/amd64], JVM[AdoptOpenJDK/OpenJDK 64-Bit Server VM/15.0.1/15.0.1+9]" }
{"type": "server", "timestamp": "2022-05-13T11:09:34,320Z", "level": "INFO", "component": "o.e.n.Node", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "JVM home [/usr/share/elasticsearch/jdk], using bundled JDK [true]" }
{"type": "server", "timestamp": "2022-05-13T11:09:34,320Z", "level": "INFO", "component": "o.e.n.Node", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "JVM arguments [-Xshare:auto, -Des.networkaddress.cache.ttl=60, -Des.networkaddress.cache.negative.ttl=10, -XX:+AlwaysPreTouch, -Xss1m, -Djava.awt.headless=true, -Dfile.encoding=UTF-8, -Djna.nosys=true, -XX:-OmitStackTraceInFastThrow, -XX:+ShowCodeDetailsInExceptionMessages, -Dio.netty.noUnsafe=true, -Dio.netty.noKeySetOptimization=true, -Dio.netty.recycler.maxCapacityPerThread=0, -Dio.netty.allocator.numDirectArenas=0, -Dlog4j.shutdownHookEnabled=false, -Dlog4j2.disable.jmx=true, -Djava.locale.providers=SPI,COMPAT, -Xms1g, -Xmx1g, -XX:+UseG1GC, -XX:G1ReservePercent=25, -XX:InitiatingHeapOccupancyPercent=30, -Djava.io.tmpdir=/tmp/elasticsearch-5442880206354287143, -XX:+HeapDumpOnOutOfMemoryError, -XX:HeapDumpPath=data, -XX:ErrorFile=logs/hs_err_pid%p.log, -Xlog:gc*,gc+age=trace,safepoint:file=logs/gc.log:utctime,pid,tags:filecount=32,filesize=64m, -Des.cgroups.hierarchy.override=/, -XX:MaxDirectMemorySize=536870912, -Des.path.home=/usr/share/elasticsearch, -Des.path.conf=/usr/share/elasticsearch/config, -Des.distribution.flavor=default, -Des.distribution.type=docker, -Des.bundled_jdk=true]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,187Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [aggs-matrix-stats]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,187Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [analysis-common]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,188Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [constant-keyword]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,188Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [flattened]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,189Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [frozen-indices]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,189Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [ingest-common]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,189Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [ingest-geoip]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,190Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [ingest-user-agent]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,190Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [kibana]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,190Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [lang-expression]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,191Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [lang-mustache]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,191Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [lang-painless]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,191Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [mapper-extras]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,197Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [mapper-version]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,197Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [parent-join]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,198Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [percolator]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,203Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [rank-eval]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,203Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [reindex]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,203Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [repositories-metering-api]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,204Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [repository-url]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,204Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [search-business-rules]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,204Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [searchable-snapshots]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,204Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [spatial]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,205Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [transform]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,205Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [transport-netty4]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,205Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [unsigned-long]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,206Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [vectors]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,206Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [wildcard]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,206Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-analytics]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,212Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-async]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,213Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-async-search]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,213Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-autoscaling]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,213Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-ccr]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,214Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-core]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,214Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-data-streams]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,214Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-deprecation]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,214Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-enrich]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,215Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-eql]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,215Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-graph]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,215Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-identity-provider]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,255Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-ilm]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,255Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-logstash]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,256Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-ml]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,256Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-monitoring]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,256Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-ql]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,256Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-rollup]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,258Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-security]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,258Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-sql]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,259Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-stack]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,259Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-voting-only-node]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,259Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "loaded module [x-pack-watcher]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,260Z", "level": "INFO", "component": "o.e.p.PluginsService", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "no plugins loaded" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,463Z", "level": "INFO", "component": "o.e.e.NodeEnvironment", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "using [1] data paths, mounts [[/ (overlay)]], net usable_space [18.6gb], net total_space [29.9gb], types [overlay]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,463Z", "level": "INFO", "component": "o.e.e.NodeEnvironment", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "heap size [1gb], compressed ordinary object pointers [true]" }
{"type": "server", "timestamp": "2022-05-13T11:09:45,691Z", "level": "INFO", "component": "o.e.n.Node", "cluster.name": "docker-cluster", "node.name": "68a4a5ec8af7", "message": "node name [68a4a5ec8af7], node ID [IaurLHJbQR6k5SV9fI3veA], cluster name [docker-cluster], roles [transform, master, remote_cluster_client, data, ml, data_content, data_hot, data_warm, data_cold, ingest]" }

11:09:58,269  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [LifecycleService] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] [127.0.0.1]:5701 is SHUTTING_DOWN
11:09:58,275  WARN |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [Node] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Terminating forcefully...
11:09:58,276  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [Node] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Shutting down connection manager...
11:09:58,292  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [Node] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Shutting down node engine...
11:09:58,324  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [NodeExtension] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Destroying node NodeExtension.
11:09:58,324  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [Node] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] Hazelcast Shutdown is completed in 49 ms.
11:09:58,325  INFO |given_clientWithWrongPassword_whenWriteToElasticSink_thenFailWithAuthenticationException| - [LifecycleService] main - [127.0.0.1]:5701 [dev] [5.2-SNAPSHOT] [127.0.0.1]:5701 is SHUTDOWN
```

</details>

--- a/extensions/elasticsearch/elasticsearch-7/src/test/java/com/hazelcast/jet/elastic/ElasticSupport.java
+++ b/extensions/elasticsearch/elasticsearch-7/src/test/java/com/hazelcast/jet/elastic/ElasticSupport.java
@@ -18,8 +18,6 @@ package com.hazelcast.jet.elastic;
 
 import com.hazelcast.function.SupplierEx;
 import com.hazelcast.jet.impl.util.Util;
-import org.apache.http.HttpHost;
-import org.elasticsearch.client.RestClient;
 import org.elasticsearch.client.RestClientBuilder;
 import org.testcontainers.elasticsearch.ElasticsearchContainer;
 import org.testcontainers.utility.DockerImageName;
@@ -63,8 +61,10 @@ public final class ElasticSupport {
     }
 
     public static SupplierEx<RestClientBuilder> elasticClientSupplier() {
-        String address = elastic.get().getHttpHostAddress();
-        return () -> RestClient.builder(HttpHost.create(address));
+        ElasticsearchContainer container = elastic.get();
+        String containerHost = container.getHost();
+        Integer port = container.getMappedPort(PORT);
+        return () -> client(containerHost, port);
     }
 
     public static SupplierEx<RestClientBuilder> secureElasticClientSupplier() {

--- a/extensions/elasticsearch/elasticsearch-7/src/test/java/com/hazelcast/jet/elastic/ElasticSupport.java
+++ b/extensions/elasticsearch/elasticsearch-7/src/test/java/com/hazelcast/jet/elastic/ElasticSupport.java
@@ -24,6 +24,7 @@ import org.elasticsearch.client.RestClientBuilder;
 import org.testcontainers.elasticsearch.ElasticsearchContainer;
 import org.testcontainers.utility.DockerImageName;
 
+import java.time.Duration;
 import java.util.function.Supplier;
 
 import static com.hazelcast.jet.elastic.ElasticClients.client;
@@ -36,7 +37,8 @@ public final class ElasticSupport {
 
     // Elastic container takes long time to start up, reusing the container for speedup
     public static final Supplier<ElasticsearchContainer> elastic = Util.memoize(() -> {
-        ElasticsearchContainer elastic = new ElasticsearchContainer(ELASTICSEARCH_IMAGE);
+        ElasticsearchContainer elastic = new ElasticsearchContainer(ELASTICSEARCH_IMAGE)
+                .withStartupTimeout(Duration.ofMinutes(2L));
         elastic.start();
         Runtime.getRuntime().addShutdownHook(new Thread(elastic::stop));
         return elastic;
@@ -49,7 +51,8 @@ public final class ElasticSupport {
         ElasticsearchContainer elastic = new ElasticsearchContainer(ELASTICSEARCH_IMAGE)
                 .withEnv("ELASTIC_USERNAME", "elastic")
                 .withEnv("ELASTIC_PASSWORD", "SuperSecret")
-                .withEnv("xpack.security.enabled", "true");
+                .withEnv("xpack.security.enabled", "true")
+                .withStartupTimeout(Duration.ofMinutes(2L));
 
         elastic.start();
         Runtime.getRuntime().addShutdownHook(new Thread(elastic::stop));
@@ -66,10 +69,10 @@ public final class ElasticSupport {
 
     public static SupplierEx<RestClientBuilder> secureElasticClientSupplier() {
         ElasticsearchContainer container = elastic.get();
-        String containerIp = container.getContainerIpAddress();
+        String containerHost = container.getHost();
         Integer port = container.getMappedPort(PORT);
 
-        return () -> client("elastic", "SuperSecret", containerIp, port);
+        return () -> client("elastic", "SuperSecret", containerHost, port);
     }
 
 }

--- a/extensions/elasticsearch/elasticsearch-7/src/test/java/com/hazelcast/jet/elastic/pipeline/CommonElasticSinksPipeline.java
+++ b/extensions/elasticsearch/elasticsearch-7/src/test/java/com/hazelcast/jet/elastic/pipeline/CommonElasticSinksPipeline.java
@@ -80,6 +80,7 @@ public final class CommonElasticSinksPipeline {
                 .clientFn(elasticSupplier)
                 .bulkRequestFn(() -> new BulkRequest().setRefreshPolicy(WriteRequest.RefreshPolicy.IMMEDIATE))
                 .mapToRequestFn((TestItem item) -> new UpdateRequest(index, item.getId()).doc(item.asMap()))
+                .retries(0)
                 .build();
 
         p.readFrom(TestSources.items(items))

